eventloop_windows.go

```diff
@@ -180,6 +184,10 @@ func (el *eventloop) loopTicker() {
 
 func (el *eventloop) loopError(c *stdConn, err error) (e error) {
 	defer func() {
+		if c.conn == nil {
```

Comment:
ditto

Reply:
Can this check be covered? if not, I prefer to remove it.

Reply:
Yes I'll do that.